Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandConfigurationView.swift

```diff
@@ -294,19 +312,23 @@ private struct RenameButton: View {
             proposedNewTitle = title
             alertIsShowing = true
         } label: {
-            Text("Rename")
+            Text.rename
         }
         .buttonStyle(.bordered)
         .buttonBorderShape(.capsule)
         .font(.subheadline)
         .fontWeight(.semibold)
-        .alert("Enter a name", isPresented: $alertIsShowing) {
-            TextField("Enter area name", text: $proposedNewTitle)
-            Button("OK", action: submitNewTitle)
+        .alert(enterName, isPresented: $alertIsShowing) {
+            TextField(text: $proposedNewTitle, prompt: areaName) {}
+            Button(String.ok, action: submitNewTitle)
                 .disabled(!proposedTitleIsValid)
```

Comment:
I know this is slightly off topic, and could be a change for a different PR, but should OK button be the default action ("The default button is shown with greater prominence. You can influence the default button by assigning it the `KeyboardShortcut/defaultAction` keyboard shortcut.")?

```suggestion
            Button(String.ok, action: submitNewTitle)
                .disabled(!proposedTitleIsValid)
                .keyboardShortcut(.defaultAction)
```

Reply:
I checked with our designer and the OK button should be the default action.

Reply:
Something like this is easily overlooked, but it is a nice quality of life improvement for users. Not only does the button become more prominent visually, but it also becomes actionable via the keyboard (return key).